api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/CardClient.java

```diff
@@ -49,4 +51,21 @@ public CardResponse requestNewCard(CardRequest cardRequest) {
             throw new CardClientException(e, cardsUri);
         }
     }
+
+    public DepositFundsResponse depositFundsToCard(String cardAccountId, DepositFundsRequest depositRequest) {
+        String uri = String.format("%s/%s/deposit", cardsUri, cardAccountId);
+        try {
+            ResponseEntity<DepositFundsResponse> response = restTemplate.postForEntity(uri, depositRequest, DepositFundsResponse.class);
+            if (response.getStatusCode() != HttpStatus.OK) {
```

Comment:
could we move this into method like checkResponse(). That way this method's main code block is just three lines that read really easily.